packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx

```diff
@@ -31,32 +31,40 @@ const PatientChart: React.FC = () => {
     changeWorkspaceContext(patientUuid);
   }, [patientUuid]);
 
+  const leftNavBasePath = useMemo(() => spaBasePath.replace(':patientUuid', patientUuid), [patientUuid]);
+  useEffect(() => {
+    setLeftNav({ name: 'patient-chart-dashboard-slot', basePath: leftNavBasePath });
+    return () => unsetLeftNav('patient-chart-dashboard-slot');
+  }, [leftNavBasePath]);
+
   return (
     <main className={`omrs-main-content ${styles.chartContainer}`}>
-      {isLoadingPatient ? (
-        <Loader />
-      ) : (
-        <>
-          <div
-            className={`${styles.innerChartContainer} ${
-              windowSize.size === 'normal' && active ? styles.closeWorkspace : styles.activeWorkspace
-            }`}
-          >
-            <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
-            <aside>
-              <ExtensionSlot extensionSlotName="patient-header-slot" state={state} />
-              <ExtensionSlot extensionSlotName="patient-info-slot" state={state} />
-            </aside>
-            <div className={styles.grid}>
-              <div className={styles.chartReview}>
-                <ChartReview {...state} view={view} />
-                <WorkspaceNotification />
+      <>
+        <div
+          className={`${styles.innerChartContainer} ${
+            windowSize.size === 'normal' && active ? styles.closeWorkspace : styles.activeWorkspace
+          }`}
+        >
+          <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
+          {isLoadingPatient ? (
```

Comment:
Realistically, I want to get rid of this `<Loader />` component here at all. There's not really any state that we're passing to child objects aside from "`usePatient()` resolved". Currently whatever renders in the `patient-header-slot` and the conditions app on the chart seem to be broken without waiting for that, but that should be fixable at the level of those components.